client/src/main/logic/clientCookies.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * This file is meant to be used as one centralized location to handle
+ * the cookies for the client. That way the type can be managed and we can
+ * keep track of which ones are available.
+ */
+
+import Cookies from 'js-cookie';
+
+/**
+ * Gets a particular cookie.
+ *
+ * @param {ClientCookies} cookieName the type of cookie to retrieve
+ * @returns {string} the value of the cookie or an empty string if the cookie
+ * wasn't set
+ */
+export function getCookie(cookieName: ClientCookies): string {
+  const returnedCookie = Cookies.get(cookieName);
+  if (returnedCookie) {
+    return returnedCookie;
+  }
+  return '';
+}
+
+/**
+ * The enum containing the different cookies that are managed by the client.
+ */
+export enum ClientCookies {
+  fontSize = 'fontSize',
+}
+
+export function getCookies(): string {
+  // eslint-disable-next-line
+  console.log(document.cookie);
```

Comment:
Will have to be removed when the time comes

Reply:
This whole function was removed. Thank you for catching this! This was just laziness that I didn't see it before submitting the pull request.